Jenkinsfile.nightly

```diff
@@ -102,14 +102,9 @@ pipeline {
 
         stage('Kubernetes tests') {
             steps {
-                parallel(
-                    "K8s-1.7":{
-                        sh 'cd ${TESTDIR}; K8S_VERSION=1.7 ginkgo --focus=" K8s*" -v'
-                    },
-                   "K8s-1.8":{
-                        sh 'cd ${TESTDIR}; K8S_VERSION=1.8 ginkgo --focus=" K8s*" -v'
-                    }
-                )
+               "K8s-1.8":{
```

Comment:
Is not a map, this does not compile. 
https://jenkins.cilium.io/job/Cilium-PR-Nightly-Tests-All/175/console

Reply:
@eloycoto Fixed